renovate/dependencyDashboard.js

```diff
@@ -0,0 +1,30 @@
+export class Dependency {
+  constructor(name, version) {
+    this.name = name;
+    this.version = version;
+  }
+}
+
+export const getDependenciesForRepo = ({ octokit }, repo) => {
+  return octokit
+    .request(repo.issuesUrl)
+    .then((response) => {
+      const dependencies = response
+        .data
+        .find((issue) => issue.user.login === "renovate[bot]" && issue.pull_request === undefined)
+        ?.body
+        ?.split(/\r?\n|\r|\n/g)
+        ?.map((line) => {
+          const match = line.match(/`(\S+?) (.*)`/);
```

Comment:
I think it would be worth extracting this regex to a const so it can be named to help aid the reader's understanding.

Reply:
I've put both of the regexes into `const`s with a comment for each explaining their purpose.